.github/workflows/stage_release.yaml

```diff
@@ -111,6 +111,13 @@ jobs:
           gpg-private-key: ${{ secrets.KROXYLICIOUS_RELEASE_PRIVATE_KEY }} # Value of the GPG private key to import
           overwrite-settings: true
 
+      - name: Setup Minikube
+        uses: manusa/actions-setup-minikube@v2.13.1
+        with:
+          minikube version: 'v1.35.0'
```

Comment:
are these changes be updated by dependabot? or should we add renovate for both?

Reply:
Workflows are updated by dependabot.  There's nothing to update the minikube version AFAIK.